static/js/click_handlers.js

```diff
@@ -747,6 +747,11 @@ export function initialize() {
             return;
         }
 
+        // The dropdown menu needs to process clicks to open and close.
+        if ($target.parents(".stream_selection_dropdown").length > 0) {
+            return;
+        }
```

Comment:
I'm a bit puzzled that this click handler would be called before the localized click handlers for the individual widget in the browser's precedence.

Reply:
We can't prevent default in localized click handlers because the dropdown menu is bootstrap :(

Reply:
Though hm yeah I'm curious why it went through here before going through bootstrap's handler.